chain/client/src/client.rs

```diff
@@ -480,12 +470,10 @@ impl Client {
             let shard_uid = shard_id_to_uid(self.epoch_manager.as_ref(), shard_id, &epoch_id)?;
 
             if block.header().height() == chunk_header.height_included() {
-                if self.shard_tracker.cares_about_shard_this_or_next_epoch(
-                    Some(me),
-                    block.header().prev_hash(),
-                    shard_id,
-                    false,
```

Comment:
ditto

Reply:
Good catch, yes I had noticed this as well, but in my opinion this is probably an oversight where is_me was supposed to be true. In any case, having is_me as true is less restrictive than having it as false.